scripts/feature_tests/setup_env.sh

```diff
@@ -5,8 +5,9 @@ set -x
 M3PATH="$(dirname "$(readlink -f "${0}")")/../.."
 
 if [[ "${1}" == "ug" ]]; then
-  export CAPM3RELEASE="v0.3.2"
-  export CAPIRELEASE="v0.3.4"
+  # shellcheck disable=SC1091
+  # shellcheck source="$M3PATH/scripts/feature_tests/upgrade/upgrade_vars.sh"
+  source "$M3PATH/scripts/feature_tests/upgrade/upgrade_vars.sh"
```

Comment:
is this needed if you define the variables in the makefile ?

Reply:
yes, based on local metal3-dev-env tests